sites/plasticsmachinerymagazine.com/server/routes/published-content.js

```diff
@@ -1,6 +1,6 @@
-const webinars = require('@endeavor-business-media/package-shared/templates/published-content/webinars');
```

Comment:
Instead of overwriting the entire template to change the title, update `@endeavor-business-media/package-shared/templates/published-content/webinars` to:

```marko
import defaultValue from "@base-cms/marko-core/utils/default-value";

$ const title = defaultValue(data.title, "Webcasts");

<shared-published-content-page-list-layout title=title>
  <@query-params content-types=["Webinar"] sort-field="startDate" sort-order="desc" />
</shared-published-content-page-list-layout>

```

Then, on L7, change this `app.get('/webinars', (_, res) => { res.marko(webinars); });` to:
```js
app.get('/webinars', (_, res) => { res.marko(webinars, { title: 'Webinars' }); });
```